transcendence-app/src/user/user.service.ts

```diff
@@ -14,6 +18,20 @@ export class UserService {
   retrieveUserWithId(id: string) {
     return this.users.find((user) => user.id === id) || null;
   }
+  retrieveUsers(queryDto: UsersPaginationQueryDto) {
+    if (queryDto.limit || queryDto.offset) {
+      let ret: string[] = [];
```

Comment:
> Variable initializer is redundant

You don't need to initialize with [] when the variable is already declared as an array.